lib/Server.js

```diff
@@ -342,10 +342,11 @@ class Server {
 
   setupStaticFeature() {
     const contentBase = this.options.contentBase;
+    const contentBasePublicPath = this.options.contentBasePublicPath;
```

Comment:
```
const contentBasePublicPath = this.options.contentBasePublicPath || '*';
```

Reply:
added default value in `utils/normalizeOptions`